lib/vector-core/Cargo.toml

```diff
@@ -65,6 +65,7 @@ vector-config = { path = "../vector-config" }
 vector-config-common = { path = "../vector-config-common" }
 vector-config-macros = { path = "../vector-config-macros" }
 vrl.workspace = true
+uuid = { version = "1.9.1", features = ["v4", "serde"] }
```

Comment:
How about v7? I think it may be useful to use the time-sortable property of those

And more: https://uuid7.com/

Reply:
Good call, that could really be useful for relatively free.

Reply:
Also, I see other parts of Vector are using this crate. Could you move it to a workspace dep?